File: C# Part 1/04.Console-Input-and-Output/06.QuadraticEquation.cs
using System;

namespace QuadraticEquation
{
    class QuadraticEquation
    {
        static void Main()
        {
            double valA = Convert.ToDouble(Console.ReadLine());
            double valB = Convert.ToDouble(Console.ReadLine());
            double valC = Convert.ToDouble(Console.ReadLine());
            double valX1, valX2;

            double discriminant = Math.Pow(valB, 2) - (4 * valA * valC);

            if (discriminant == 0)
                Console.WriteLine("{0:F2}", -(valB / (2 * valA)));
            else if (discriminant > 0)
            {
                valX1 = -(valB / (2 * valA)) + (Math.Sqrt(discriminant)) / (2 * valA);
                valX2 = -(valB / (2 * valA)) - (Math.Sqrt(discriminant)) / (2 * valA);
                if (valX1 < valX2) Console.WriteLine("{0:F2}\n{1:F2}", valX1, valX2);
                else Console.WriteLine("{0:F2}\n{1:F2}", valX2, valX1);
            }
            else
                Console.WriteLine("no real roots");
        }
    }
}

File: JavaScript Fundamentals/05. Conditional Statements/06_QuadraticEquation.js
function solve(args) {

    var a = +(args[0]),
        b = +(args[1]),
        c = +(args[2]);

    var discriminant = (b * b) - (4 * (a * c));

    if(discriminant > 0){
        var x2 = (-b + Math.sqrt(discriminant))/ (2 * a);
        var x1 = (-b - Math.sqrt(discriminant))/ (2 * a);

        console.log('x1=' + x1.toFixed(2) + '; ' + 'x2=' + x2.toFixed(2));
    }else if(discriminant === 0){
        var x = -b/(2 * a);

         console.log('x1=x2=' + x.toFixed(2));
    }else{
        console.log('no real roots');
    }
}

// solve(['2', '5', '-3']);
solve(['-0.5', '4', '-8']);

